source/_docs/guides/drupal-8-commerce.md

```diff
@@ -0,0 +1,122 @@
+ - [Build Tools](/docs/guides/build-tools)
+
+
+In addition to Pantheon, you will need accounts at:
+
+ - [GitHub](https://github.com){.external}
+ - [CircleCI](https://circleci.com){.external}
+
+
+1.  Follow the [Before You Begin](/docs/guides/build-tools/#before-you-begin) section of the Build Tools guide to install Composer, Terminus, and the Terminus Build Tools plugin on your local computer, and create machine tokens for [GitHub](https://help.github.com/articles/creating-an-access-token-for-command-line-use/){.external} and [CircleCI](https://circleci.com/account/api){.external}. Export the tokens to your current terminal session, as described below.
+
+2.  This guide uses several variables in example [Terminus](/docs/terminus) commands. This lets you copy and paste without needing to change the variable. For this to work, you must first export the variables in your local terminal session:
+
+    ```bash
+    export SITENAME=yoursitenamehere
+    export GITHUB_TOKEN=yourgithubtokenhere
+    export CIRCLE_TOKEN=yourcirclecitokenhere
+    ```
+
+    - `SITENAME` will be used as the machine names of the Pantheon site and the GitHub repo created in this process
+    - `GITHUB_TOKEN` lets Terminus interact with your GitHub account to create the repository
+    - `CIRCLE_TOKEN` is used to configure CircleCI to push to Pantheon any time a push is made to the GitHub repo
+
+## Create a New Drupal 8 Site
+
+1.  Using the Terminus Build Tools plugin, create a new Drupal 8 site from the Pantheon [Example Drops 8 Composer](https://github.com/pantheon-systems/example-drops-8-composer) repository on GitHub:
+
+    ```bash
+    terminus build:project:create pantheon-systems/example-drops-8-composer $SITENAME
+    ```
+
+    At this point *do not* go to the web interface to continue installation.
+
+2.  You now have a repository on GitHub containing your new site. Clone a local copy to your `projects` folder:
+
+    ```bash
+    cd ~/projects
+    git clone git@github.com:username/$SITENAME.git
+    ```
+
+    Remember to replace `username` with your GitHub username.
+
+## Install Drupal Commerce
+
+1. Move into the local repository for your site:
+
+        cd $SITENAME
+
+2. Use Composer to install the [Commerce Installation Profile](https://github.com/drupalcommerce/commerce_base){.external}:
+
+   ```bash
+   composer config repositories.commerce_base vcs https://github.com/drupalcommerce/commerce_base
+   composer require "drupalcommerce/commerce_base dev-8.x-1.x" "drupal/commerce:~2.0"  "drupal/admin_toolbar:~1.0"  "drupal/swiftmailer:~1.0"
+   ```
+
+    <div class="alert alert-info" role="alert">
+    <h4 class="info">Note</h4>
+    <p markdown="1">The commands above include dependencies required but not listed for `commerce_base`.  simpler once [a few Composer changes are made within the Commerce Installation Profile itself](https://github.com/drupalcommerce/commerce_base/pull/7){.external}.</p>
```

Comment:
Fragment:  "simpler once [a few Composer changes are made within the Commerce Installation Profile itself]"

Reply:
Good catch! Partial edit from an earlier review. I meant to come back to that...